src/commoncode/text.py

```diff
@@ -181,10 +184,12 @@ def toascii(s, translit=False):
         else:
             converted = unicodedata.normalize('NFKD', s).encode('ascii', 'ignore')
     except:
-        converted = str(s.decode('ascii', 'ignore'))
+        if py2:
+            converted = compat.unicode(s.decode('ascii', 'ignore'))
+        else:
+            converted = compat.unicode(s)
```

Comment:
You are no longer normalizing to ASCII on Python3.

Reply:
Normalising to ascii on Py3 throwing an error